src/System.Threading.Overlapped/src/System.Threading.Overlapped.csproj

```diff
@@ -41,8 +37,14 @@
       <Link>Common\NotImplemented.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'=='netstandard'">
+  <ItemGroup Condition="'$(TargetGroup)'=='netstandard' AND '$(TargetsWindows)' == 'true'">
     <TargetingPackReference Include="System.Private.CoreLib" />
+    <Reference Include="System.Private.CoreLib" />
```

Comment:
I noticed that all of the partial facades require this redundant ref to trigger https://github.com/dotnet/buildtools/blob/7b3991e4bba0b86945a06470b313f9183e744e25/src/Microsoft.DotNet.Build.Tasks/PackageFiles/packageresolve.targets#L154.

@weshaggard mentioned we should just turn that on by default.  I've filed https://github.com/dotnet/corefx/issues/14669 to clean these up.